src/planner/select_plan_test.go

```diff
@@ -284,13 +284,12 @@ func TestSelectPlanDatabaseIsNull(t *testing.T) {
 
 func TestSelectUnsupportedPlan(t *testing.T) {
 	querys := []string{
-		"select * from A as A where id in (select id from B)",
+		"select * from A as A1 where id in (select id from B)",
 		"select distinct(b) from A",
 		"select A.id from A join B on B.id=A.id",
 		"select id, rand(id) from A",
 		"select id from A order by b",
 		"select id from A limit x",
-		"select 1",
```

Comment:
Why remove this test case?

Reply:
The sql 'select 1' is used  in the test function "TestSelectUnsupportedPlan()" and want a result  "Table 'dual' doesn't exist (errno 1146) (sqlstate 42S02)",  but we had surpport operation on  dual,  the  commit log surpporting dual operation is : [commit log](https://github.com/radondb/radon/commit/f44a8f67c2553703af36ad302ae5fd7fcda52039)